demo/index.html

```diff
@@ -2,16 +2,15 @@
 <html>
 <head>
     <title>jVectorMap demo</title>
-    <script src="jquery.js"></script>
-
     <link rel="stylesheet" href="jquery-jvectormap-2.0.4.css" type="text/css" media="screen"/>
 
 
+    <script src="jquery.js"></script>
```

Comment:
Instead of manually including the jquery, let's use the hosted libraries by Google: https://developers.google.com/speed/libraries/#jquery

Use version `1.11.3` to make sure the plugin keeps supporting IE :smiling_imp: